cookiecutter.json

```diff
@@ -1,5 +1,11 @@
 {
   "app_name": "default-app-name",
   "use_logrus": ["yes", "no"],
-  "web_variant": ["yes", "no"]
+  "variant": ["API only", "Web only", "Both"],
+  "css_addon": ["None", "Bootstrap", "Tailwinds"],
+
+  "_api_variant": "n",
```

Comment:
Should we change to use `yes/no` to make it consistent with other options?

Reply:
ok for me. @gutakk what do you think? :)

Reply:
Yes, I think we should go with `yes/no` to make it consistent with `use_logrus`